packages/sign-client/canary/canary.spec.ts

```diff
@@ -0,0 +1,53 @@
+import "mocha";
+import {
+  initTwoClients,
+  testConnectMethod,
+  deleteClients,
+} from "./../test/shared";
+import CloudWatch from 'aws-sdk/clients/cloudwatch';
+
+const environment = process.env.ENVIRONMENT || 'dev';
+
+describe("Canary", function() {
+  describe("HappyPath", function() {
+    // TODO: implement a test that depicts
+    // the happy case better
+    it("connects", async function() {
+      const clients = await initTwoClients();
+      await testConnectMethod(clients);
+      deleteClients(clients);
+    });
+  });
+  afterEach(function (done) {
+    const cloudwatch = new CloudWatch({region: 'eu-central-1'});
```

Comment:
Can we move this callback out into its own function instead of inlining it, i.e.

```js
const uploadToCloudwatch = function (done) {...}

// ...

afterEach(uploadToCloudwatch)

```